helper-cli/build.gradle.kts

```diff
@@ -56,6 +56,7 @@ repositories {
 
 dependencies {
     implementation(project(":analyzer"))
+    implementation(project(":cli"))
```

Comment:
This dependency between CLI modules is something we got rid of a while go, and I'd prefer to keep it like that. Either by moving shared code to a common module, or duplication (small amounts of) code in the worst case.

Reply:
What is this needed for anyway?